python/example/single_cell_cable.py

```diff
@@ -0,0 +1,213 @@
+        decor.set_property(Vm=self.Vm)
+        decor.set_property(cm=self.cm)
+        decor.set_property(rL=self.rL)
+
+        decor.paint('"cable"',
+                    arbor.mechanism(f'pas/e={self.Vm}', {'g': self.g}))
+
+        decor.place('"start"', arbor.iclamp(args.stimulus_start,
+                                            args.stimulus_duration,
+                                            args.stimulus_amplitude))
+
+        policy = arbor.cv_policy_max_extent(self.cv_policy_max_extent)
+        decor.discretization(policy)
+
+        cell = arbor.cable_cell(tree, labels, decor)
+
+        return cell
+
+    def probes(self, gid):
+        assert gid == 0
+        return self.the_probes
+
+    def global_properties(self, kind):
+        return self.the_props
+
+
+def get_rm(g):
+    """Return membrane resistivity in Ohm*m^2
+    g -- membrane conductivity in Siemens/m^2
+    """
+    return 1/g
+
+
+def get_taum(cm, rm):
+    """Return membrane time constant in seconds
+    cm -- membrane capacitance in F/m^2
+    rm -- membrane resistivity in Ohm*m^2
+    """
+    return cm*rm
+
+
+def get_lambdam(a, rm, rL):
+    """Return electronic length in m
+    a -- cable radius in m
+    rm -- membrane resistivity in Ohm*m^2
+    rL -- axial resistivity in Ohm*m
+    """
+    return np.sqrt(a*rm/(2*rL))
+
+
+def get_vcond(lambdam, taum):
+    """Return conductance velocity in m/s
+    lambda -- electronic length in m
+    taum -- membrane time constant
+    """
+    return 2*lambdam/taum
+
+
+def get_tmax(data):
+    """Return time of maximum of membrane voltage"""
+    return data[np.argmax(data[:, 1])][0]
+
+
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser(description='Cable')
+
+    parser.add_argument(
+        '--Vm', help="membrane leak potential in mV", type=float, default=-65)
+    parser.add_argument(
+        '--length', help="cable length in micrometer", type=float, default=1000)
+    parser.add_argument(
+        '--radius', help="cable radius in micrometer", type=float, default=1)
+    parser.add_argument(
+        '--cm', help="membrane capacitance in F/m^2", type=float, default=0.01)
+    parser.add_argument(
+        '--rL', help="axial resistivity in Ohm*centimeter", type=float, default=90)
+    parser.add_argument(
+        '--g', help="membrane conductivity in Siemens/cm^2", type=float, default=0.001)
+
+    parser.add_argument('--stimulus_start',
+                        help="start of stimulus in ms", type=float, default=10)
+    parser.add_argument('--stimulus_duration',
+                        help="duration of stimulus in ms", type=float, default=0.1)
+    parser.add_argument('--stimulus_amplitude',
+                        help="amplitude of stimulus in nA", type=float, default=1)
+
+    parser.add_argument('--cv_policy_max_extent',
+                        help="maximum extent of control volume in micrometer", type=float, default=10)
+
+    args = parser.parse_args()
```

Comment:
A few comments here would to explain what is going on here would also be really helpful for the tutorial.